services/galley/test/integration/API/Util.hs

```diff
@@ -938,12 +955,16 @@ getConvs u r s = do
       . zType "access"
       . convRange r s
 
-listConvs :: (MonadIO m, MonadHttp m, HasGalley m) => UserId -> ListConversations -> m ResponseLBS
+listConvs ::
+  (MonadIO m, MonadHttp m, MonadReader TestSetup m) =>
+  UserId ->
+  ListConversations ->
+  m ResponseLBS
 listConvs u req = do
-  g <- viewGalley
+  g <- view tsUnversionedGalley
   post $
     g
-      . path "/conversations/list/v2"
+      . path "/v1/conversations/list/v2"
```

Comment:
Unversioned in addition to being at v1? Is there no function `apiVersion` in Galley tests? I think it'd be worth it creating a small test package that has utilities like `apiVersion` instead of copy-pasting it several times.